src/shared/utils.ts

```diff
@@ -506,3 +507,23 @@ export function getPiscinaStats(piscina: Piscina): Record<string, number> {
         'runTime.p50': piscina.runTime.p50,
     }
 }
+
+export function pluginConfigIdFromStack(
+    stack: string,
+    pluginConfigSecretLookup: Map<string, PluginConfigId>
+): PluginConfigId | void {
+    const regexp = /at __[a-zA-Z0-9]+__PluginConfig_([0-9]+)_([0-9a-f]+) \(vm\.js\:/
```

Comment:
Could we get an example in a comment for future reference, so that we can easily know what this matches?

Reply:
Clarified it a bit.

The benchmarks test is indeed annoyingly flakey. It'll bother me enough to look at it soon enough :).

Reply:
great, thanks!